sample-project/App.fsproj

```diff
@@ -10,14 +10,14 @@
     <ItemGroup>
         <PackageReference Include="Fable.Browser.Dom" Version="2.14.0" />
         <PackageReference Include="Fable.Core" Version="4.3.0" />
-        <PackageReference Include="Nojaf.Fable.React" Version="0.0.2" />
+        <PackageReference Include="Feliz" Version="2.9.0" />
```

Comment:
update to standard fable Feliz remove Nojaf.Fable.React experimental package from sample